app.rb

```diff
@@ -1,163 +1,20 @@
-require_relative './classroom'
+require_relative './book'
+require_relative './person'
 require_relative './student'
 require_relative './teacher'
-require_relative './book'
 require_relative './rental'
+require_relative './modules/show_books'
+require_relative './modules/show_person'
+require_relative './modules/create_person'
+require_relative './modules/create_book'
+require_relative './modules/create_rental'
+require_relative './modules/list_rental'
```

Comment:
- I currently have this error when I try to run your app. I guess you have not imported your files correctly, Please double-check that you are using the right paths when requiring a module. Please fix this 🙏 
- Also ensure your app runs without error on your end before requesting a review of your work 👍 👍 

<img width="724" alt="ethethethe" src="https://user-images.githubusercontent.com/73607512/211413535-a2737e04-a3ed-4220-b03c-1dba28a45a17.PNG">